Navbar styling

johnsciutto opened the pull request.

johnsciutto committed:
Almost done with styling. Remaining issue: focus

When the search-bar is focused on, a red outline appears. Troubleshooting with
the team.

johnsciutto committed:
Change to individual exports

johnsciutto committed:
Add main colors to theme. Move search icon

johnsciutto committed:
Fix: Console warnings about invalid props

johnsciutto committed:
Moved Navbar to /src/components.

johnsciutto committed:
Add navigation to /settings from navbar

johnsciutto committed:
Styling changes with prettier

johnsciutto committed:
Changed method from redirect to push

HenryZou1 reviewed (approved):
good job

jessdesr commented on client/src/components/Navbar.js:
```diff
@@ -0,0 +1,84 @@
+  const classes = useStyles();
+  const history = useHistory();
+
+  const handleClickSettings = () => history.push("/settings");
```
instead of pushing onto the history, it might be better to use react-router's LInk component https://reactrouter.com/web/api/Link

johnsciutto replied:
Fixed!

johnsciutto committed:
Change navigation to settings using link

johnsciutto committed:
Change size of search icon

johnsciutto committed:
Remove useHistory import

johnsciutto committed:
Remove useHistory import

johnsciutto committed:
Merge branch 'navbar-styling' of https://github.com/hatchways/team-banh-mi into navbar-styling

johnsciutto committed:
Changed sizes to use the theme

johnsciutto commented:
![Screenshot from 2021-02-24 06-56-22](https://user-images.githubusercontent.com/31010488/108997336-79171200-766d-11eb-9b26-d368e9f233a9.png)

johnsciutto committed:
Merged dev into navbar-styling. Solved conflicts.

johnsciutto committed:
Delete pages/navbar.js. Incorporat Navbar into Settings.

johnsciutto committed:
Fixed navbar hiding content underneath.

johnsciutto committed:
Add dynamic icon to navbar.

When in the settings page, it's a home icon that navigates back to the
dashboard. In all other pages it's a settings icon that navigates to the
settings page.

johnsciutto commented:
@jessdesr , the navbar is completely done and ready to be reviewed.
I changed the icon when on the settings page to link to dashboard (and be a "home" icon), and in all other pages it will link to settings. It made more sense.

![Screenshot from 2021-02-24 17-40-01](https://user-images.githubusercontent.com/31010488/109075646-721ced80-76c7-11eb-849b-def489695907.png)
![Screenshot from 2021-02-24 17-40-16](https://user-images.githubusercontent.com/31010488/109075648-721ced80-76c7-11eb-9523-f8972f025135.png)
